igvc_ws/src/sensors/src/sensors_webhook.py

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env python
+
+import socket
+import fcntl
+import struct
+import os
+import json as j
+
+import rospy
+import ros_api as ros
+
+import custom_msgs.msg as msg
+import std_msgs as std_msgs
+
+from flask import Flask, request, abort, Response, jsonify
+
+app = Flask(__name__)
+
+# Varialbes go here
+publisher = None # this is the publisher
+gps_location = None
+acceleration = None
+
+@app.route('/webhook')
+def webhook():
+    return "Webhooks :)"
+
+
+@app.route('/webhook/accelerometer', methods=['GET', 'POST'])
+def accel():
+    global acceleration
+    if request.method == 'POST':
+        data = request.data
+        json = j.loads(data)
+        acceleration = json
+        rospy.loginfo('/webhook/accelerometer{' +  'type: ' + str(json['type']) + ', x: ' + str(json['x']) + ', y: ' + str(json['y']) + ', z: ' + str(json['z']) + '}')
+        return 'success'
+    elif request.method == 'GET':
+        #rospy.loginfo("Getting gps location")
+        return jsonify(acceleration)
+
+
+@app.route('/webhook/gps', methods=['GET', 'POST'])
+def gps():
```

Comment:
You might have a better idea about this, but maybe return a boolean for success rather than a string, to me that seems more useful when called.